src/Standards/Generic/Sniffs/CodeAnalysis/ForLoopShouldBeWhileLoopSniff.php

```diff
@@ -57,7 +57,7 @@ public function process(File $phpcsFile, $stackPtr)
         $token  = $tokens[$stackPtr];
 
         // Skip invalid statement.
-        if (isset($token['parenthesis_opener']) === false) {
+        if (isset($token['parenthesis_opener']) === false || isset($token['parenthesis_closer']) === false) {
```

Comment:
```suggestion
        if (isset($token['parenthesis_opener'], $token['parenthesis_closer']) === false) {
```

No biggie, just a tiny simplification.